odoo/service/db.py

```diff
@@ -72,6 +73,10 @@ def _initialize_db(id, db_name, demo, lang, user_password, login='admin', countr
                 countries = env['res.country'].search([('code', 'ilike', country_code)])
                 if countries:
                     env['res.company'].browse(1).country_id = countries[0]
+                if len(country_timezones[country_code]) == 1:
+                    users = env['res.users'].search(['|', ('active','=', True), ('active','=', False)])
```

Comment:
You could use `active_test` context key to do so but I do not think it is a good idea anyway. Just leave no tz for the inactive users.